ee/tabby-webserver/src/oauth/gitlab.rs

```diff
@@ -129,6 +140,26 @@ impl OAuthClient for GitlabClient {
         Ok(email.email)
     }
 
+    async fn fetch_user_full_name(&self, access_token: &str) -> Result<String> {
+        let resp = self
+            .client
+            .get("https://gitlab.com/api/v4/user")
```

Comment:
Slightly different to `github` & `google`, we can use the same [user API](https://docs.gitlab.com/ee/api/users.html#list-current-user) to retrieve user name info (correct me if I'm wrong @boxbeam)

So I just duplicated implementation from `fetch_user_email`.